client/cypress/support/index.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands';
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```

Comment:
Given that `supportFile` is set to `false` in `cypress.json`, should we keep this file around?

I checked, and it looks like cypress only runs this file if you have

```json
"supportFile": "cypress/support/index.ts"
```

in `cypress.json`. 🤷🏻‍♂️

Reply:
Oh then we should probably fix `cypress.json`. I think we should have these default files available incase someone wants to use custom commands.

Reply:
Oh, okay! 🙂 I'll go make a suggestion to that effect.